### DISCUSSION TITLE
How to keep the same face and the same body once you've generated a "good" AI model ?

### DISCUSSION THREAD

**Discussion by GeorgeFlorian** [lllyasviel/Fooocus#2043] [Q&A]
I've just downloaded Fooocus and started playing around, but the same prompt generates a different result each time.

How can I keep the same face and the same body on a generated model and just generate a different pose, clothes, backgrounds ?

**Comment by mashb1t** [comment]
You can upload up to 4 images as image prompt to use as reference for Fooocus.
Also see https://github.com/lllyasviel/Fooocus/discussions/2036

**Reply by mashb1t** [reply]
plus tutorial in https://www.youtube.com/watch?v=ky5ZB-mqZKM
